python/dolfinx/io/gmshio.py

```diff
@@ -258,6 +268,27 @@ def model_to_mesh(
         # Sort elements by ascending dimension
         perm_sort = np.argsort(cell_dimensions)
 
+        # Check that all cells are tagged once
+        _d = model.getDimension()
+        _elementTypes, _elementTags, _nodeTags = model.mesh.getElements(dim=_d, tag=-1)
+        # assert only one type of elements
+        # assert len(_elementTypes) == 1  # NOTE: already checked in extract_topology_and_markers
+        _elementType_dim = _elementTypes[0]
+        if _elementType_dim not in topologies.keys():
+            raise RuntimeError("All cells are expected to be tagged once; none found")
+        nbcells = len(_elementTags[0])
+        nbcells_tagged = len(topologies[_elementType_dim]["entity_tags"])
+        if nbcells != nbcells_tagged:
+            e = (
```

Comment:
Do we need to introduce `e`? Can the string go straight into `RuntimeError`?

Reply:
Done